examples/dwarfdump.rs

```diff
@@ -196,6 +197,23 @@ fn dump_line<Endian>(file: &obj::File, debug_abbrev: gimli::DebugAbbrev<Endian>)
     }
 }
 
+fn dump_aranges<Endian>(file: &obj::File)
+    where Endian: gimli::Endianity
+{
+    let debug_aranges = obj::get_section(file, ".debug_aranges")
+        .expect("Does not have .debug_aranges section");
```

Comment:
We don't require the `.debug_aranges` section to be able to dump the rest of the debug info (unlike say `.debug_abbrev`) so it makes more sense to use an `if let` rather than `expect("...")` or `unwrap()`.

```rust
if let Some(debug_aranges) = obj::get_section(file, ".debug_aranges") {
    // ...
}
```